### ISSUE TITLE
mangadex doesn't seem to work

### ISSUE THREAD

**Issue by FichteFoll** [Hamuko/cum#66]
```
~ λ cum follow https://mangadex.org/title/19435/back-street-girls
==> Invalid URL "https://mangadex.org/title/19435/back-street-girls"

~ Λ cum --version
cum version 0.9 "Morino Kirin"
```
(installed from AUR a few days ago)

My system is Arch Linux, my Python version is 3.7.0.

**Comment by gargargarrick** [comment]
Same here. In addition, if I try to follow with a `/manga/` URL instead of a `/title/` one, or update or download any series from Mangadex, I get an error with the scraper:

```
  File "/usr/lib/python3.5/site-packages/cum-0.9.post1-py3.5.egg/cum/scrapers/mangadex.py", line 89, in name
    return re.search(title_re, title).group(1)
AttributeError: 'NoneType' object has no attribute 'group'
```

Most likely, API changes are responsible.

BTW, I am on Mageia Linux 6. Just in case I tried it with both Python 3.5.3 and 3.7.0,  running the latest version from git:
```
$ cum --version
cum version git-9d7fe4c "Morino Kirin"
```

**Comment by mxnemu** [comment]
This should be fixed in b39d8ac81c8f326e76d8c2e18ebeea609ec48f53 and merged in https://github.com/Hamuko/cum/pull/64

You can run the git version with `python3 -m cum.cum [update|download|get|...]`.

@gargargarrick The API changes should also be addressed.
Thanks for the reports!

**Comment by Hamuko** [comment]
New version released.
